chrome-extension/plugin/messenger.ts

```diff
@@ -22,7 +22,7 @@ class MessengerPage {
   }
 
   getUnreadLinks() : HTMLAnchorElement[] {
-    return document.querySelectorAll('li[aria-relevant="additions text"][aria-live="polite"]');
+    return Array.prototype.slice.call(document.querySelectorAll('li[aria-relevant="additions text"][aria-live="polite"]'));
```

Comment:
`querySelectorAll` returns a NodeList, not an Array.
https://hackernoon.com/htmlcollection-nodelist-and-array-of-objects-da42737181f9

Reply:
oh yeah, good call, thanks for fixing this